etc/config/java.amazon.properties

```diff
@@ -1,4 +1,4 @@
-compilers=&java8:&java9:&java10:&java11:&java12:&java13:&java14:&java15:&java16:&java17:&java18:&java19:&java20:&java21:&java22
+compilers=&java8:&java9:&java10:&java11:&java12:&java13:&java14:&java15:&java16:&java17:&java18:&java19:&java20:&java21:&java22:&java23
 compilerType=java
 versionFlag=-version
 defaultCompiler=java2102
```

Comment:
Did you mean to leave this as a default? I think 23.01 should be a default here and will update (but let me know if that's wrong!)

Reply:
Should have been 23.0.1 as default yes

Reply:
I left 21.02 as the default on purpose, since that is the latest LTS version (see https://www.oracle.com/java/technologies/java-se-support-roadmap.html).

Reply:
Thanks for getting back: we can revert my change and go back to 21.02 as the default! Sorry I impatiently deployed after my assumption!

Reply:
Thanks for the prompt reply. Rero is great 😄